cmd/devp2p/internal/ethtest/suite_test.go

```diff
@@ -46,7 +46,7 @@ func TestEthSuite(t *testing.T) {
 	if err != nil {
 		t.Fatalf("could not create new test suite: %v", err)
 	}
-	for _, test := range suite.AllEthTests() {
+	for _, test := range suite.Eth66Tests() {
```

Comment:
This shouldn't be 67? As technically the 66 was drop

(As a note, I really think that we shot our own foot increasing the number of protocol, instead of, I don't know, call the celo ones 1065, 1066 etc)

Reply:
This is Eth66, not Celo67. Just as before it was EthTests

Reply:
Yes, but shouldn't we change the suite test to be called Celo67? Because technically as being only one protocol it will be using the celo67 under the hood. (maybe we missed before the rename from AllEthTests, to AllCeloTests)

Reply:
Yes. It seems we forgot about those in the merge of a previous upstream commit. I'll fix that in a different PR.